src/api/provider.ts

```diff
@@ -78,11 +80,10 @@ export const PROVIDERS: ProviderProps[] = [
     dataEnv: "testnet",
   },
   {
-    name: "Rococo via GC",
-    url: "wss://hydradx-rococo-rpc.play.hydration.cloud",
-    indexerUrl: "https://hydradx-rococo-explorer.play.hydration.cloud/graphql",
-    squidUrl:
-      "https://squid.subsquid.io/hydradx-rococo-data-squid/v/v1/graphql",
+    name: "Paseo",
+    url: `wss://${PASEO_WS_URL}`,
+    indexerUrl: "https://explorer.hydradx.cloud/graphql",
```

Comment:
Maybe use env vars here also?

Reply:
it's added to env vars. This config we need if we use mainnet rpcs in dev env